app/controllers/application_controller.rb

```diff
@@ -782,9 +782,10 @@ def view_to_hash(view, fetch_data = false)
       target = @targets_hash[row.id] unless row['id'].nil?
 
       new_row = {
-        :id      => list_row_id(row),
-        :long_id => row['id'].to_s,
-        :cells   => []
+        :id        => list_row_id(row),
+        :long_id   => row['id'].to_s,
+        :cells     => [],
+        :clickable => params.fetch_path(:additional_options, :clickable)
```

Comment:
Does anything have to be done with the clickable setting down below on line 808?  I'm not sure why it's there.

Reply:
@Fryguy I think,  line no 808 is needed for setting url for clickable rows.